galaxy_ng/app/api/v1/filtersets.py

```diff
@@ -57,6 +57,8 @@ class LegacyRoleFilter(filterset.FilterSet):
 
     github_user = filters.CharFilter(method='github_user_filter')
     keywords = filters.CharFilter(method='keywords_filter')
+    tags = filters.CharFilter(method='tags_filter')
+    tag = filters.CharFilter(method='tags_filter')
```

Comment:
i see in the integration test that `tag` is a query param... is `tags` used by the UI also?

Reply:
The UI uses "tags", but it doesn't make contextual sense if you are filtering on a single tag. We could extend "tags" to mean a comma separated list, but there's no convention or ask for that yet.